src/Events/appsettings.json

```
@@ -3,12 +3,12 @@
     "EnableDBConnection": "true",
     "WorkspacePath": "Migration",
     "AdminConnectionString": "Host=localhost;Port=5432;Username=platform_events_admin;Password={0};Database=eventsdb",
-    "ConnectionString": "Host=localhost;Port=5432;Username=platform_events;Password={0};Database=eventsdb",
+    "ConnectionString": "Host=localhost;Port=5432;Username=platform_events;Password={0};Database=eventsdb;Include Error Detail=True",
     "EventsDbAdminPwd": "Password",
     "EventsDbPwd": "Password"
   },
   "GeneralSettings": {
-    "Hostname": "localhost:5080",
+    "BaseUri": "https://platform.localhost:5080",
```

```suggestion
    "BaseUri": "http://localhost:5080",
```
I didn't test this, but I think this is closer to the correct uri.

---

I was considering doing that change, so now I have to :)